emscripten.py

```diff
@@ -395,9 +384,9 @@ def create_module(function_table_sigs, metadata, settings,
 
 
 def write_output_file(outfile, post, module):
-  for i in range(len(module)): # do this loop carefully to save memory
```

Comment:
i think this loop intentionally did not leave a variable with chunk contents alive. here, the new variable `chunk` survives. maybe this mattered a long time ago but not now, though, but we should verify